ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -467,24 +490,25 @@ module Vendor = functor (V : VENDOR) -> struct
       ~is_pci_hidden =
     let vgpu_types = make_vgpu_types ~__context ~pci in
     (* Temporarily fall back to old nvidia module *)
-    if vendor_id = Nvidia_old.vendor_id then begin
+    info "Using new Nvidia config file";
+    (* if vendor_id = Nvidia_old.vendor_id then begin
```

Comment:
Does commenting this out have any implication for updates from an older version?
Or did we drop the Nvidia_old.vendor_id cards from the HCL?

Reply:
Actually it's not related to dropping any nvidia_old cards from HCL. Here the old means just the way to read from Nvidia config file as Nvidia changed their config file from separated to one.
So I just enable the new way to read config(b66e46cc38e40ac6f0a1d3f3576575b09f49825c).